test/unit/flash_attention/flash_attention_prefill/flash_prefill_testbed_3x.hpp

```diff
@@ -187,6 +252,27 @@ struct TestbedImpl {
   // Methods
   //
 
+  template <typename SrcT, typename DstT>
+  void convert_fp8_to_fp16(const SrcT* d_src, DstT* d_dst, size_t size) {
```

Comment:
Assuming this PR lands after https://github.com/codeplaysoftware/cutlass-sycl/pull/351/, we could use a single definition for `convert_fp8_to_fp16` which I suggested to move to `sycl_common.hpp`

Reply:
I'll rename it as `convert_dtype` & move it to `sycl_common.hpp`. Thanks!